src/test/kotlin/org/opensearch/replication/bwc/BackwardsCompatibilityIT.kt

```diff
@@ -139,6 +142,19 @@ class BackwardsCompatibilityIT : MultiClusterRestTestCase() {
                 Assertions.assertThat(getResponse.isExists).isTrue()
                 Assertions.assertThat(getResponse.sourceAsMap).isEqualTo(source)
             }, 60, TimeUnit.SECONDS)
+
+            //Check for latest retention lease when full cluster restart is done
+            if(ClusterStatus.from(System.getProperty("tests.bwcTask")) == ClusterStatus.FULL_CLUSTER_RESTART){
+                assertBusy ({
+                    val followerClusterInfo : Map<String, Any>   = OpenSearchRestTestCase.entityAsMap(follower.lowLevelClient.performRequest(Request("GET", "/")))
+                    val clusterUUID = (followerClusterInfo["cluster_uuid"] as String)
+                    assert(clusterUUID.isNotEmpty())
+                    val retentionLeaseinfo = leader.lowLevelClient.performRequest(Request("GET", "/$LEADER_INDEX/_stats/docs?level=shards"))
+                    val retentionLeaseInfoString = EntityUtils.toString(retentionLeaseinfo.entity)
+                    assertTrue(retentionLeaseInfoString.contains(clusterUUID))
```

Comment:
Let's have proper assertions for the actual retentionLeaseID for atleast one of the shards as there are no other test cases for the helper class.

Reply:
Done